docker/tls-setup/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine
+
+WORKDIR /code
+
+RUN apk update && apk add curl && apk add openssl && apk add openjdk8-jre-base && apk add bash && apk add bind-tools
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl;
+RUN chmod +x kubectl; mv kubectl /usr/local/bin/kubectl
+
+ADD gencerts.sh /code/
+
+# Create 'cassandra' user and group
+RUN sed -i "s/999/99/" /etc/group
```

Comment:
it is a hack; the alpine docker image already had a _group_ 999, so it's easier to fix that instead of updating all the other spots we use this specific id. Also, this is only one time, when we build the docker image for the tls-setup and to build certs, not for anything else.